CMakeLists.txt

```diff
@@ -145,7 +149,7 @@ if(PIE)
   set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 endif()
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)

```

Comment:
This is a part of another PR to migrate to C++14/17, right?

Reply:
This seems to be required when you add a custom type, I haven't managed to get that part of the code to compile using only C++11.